components/ww3/cime_config/config_compsets.xml

```diff
@@ -30,8 +30,8 @@
   </compset>
 
   <compset>
-    <alias>GMPAS-JRA-WW3</alias>
-    <lname>2000_DATM%JRA_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA_SGLC_WW3</lname>
+    <alias>GMPAS-JRA-1p4-2018-WW3</alias>
+    <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA_SGLC_WW3</lname>
```

Comment:
You need to designate the `JRA-1p4-2018` for the `DROF` component as well as `DATM`:

```
<lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA-1p4-2018_SGLC_WW3</lname>
```

You don't necessarily need the `2018` part in the alias, so `GMPAS-JRA1p4-WW3` might be easier (but it's up to you). That's what we did for G-cases:
https://github.com/E3SM-Project/E3SM/blob/2491723ccacd2e960502789fb5c3f90182ec462d/components/mpas-ocean/cime_config/config_compsets.xml#L56

Reply:
Very helpful, thanks. I overlooked the JRA for DROF.